namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/00-namespace.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "hmpps-book-a-video-link-dev"
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+    pod-security.kubernetes.io/enforce: restricted
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/slack-channel: "farsight-devs"
+    cloud-platform.justice.gov.uk/application: "Book a video link"
+    cloud-platform.justice.gov.uk/owner: "farsight-devs: dps-hmpps@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/hmpps-book-a-video-link.git,https://github.com/ministryofjustice/hmpps-book-a-video-link-api.git"
```

Comment:
@timharrison-moj are we happy we these Git repo names?

Reply:
Yes, look fine to me. Very much like the original, with an added 'a'
The thing to bear in mind - we want to deploy this under a different name / cert - alongside the original - and then switch the DNS to rollout.